API.md

```diff
@@ -17,14 +17,18 @@ Schwifty takes the following registration options,
 Returns the knex instance used by the current plugin. In other words, this returns the knex instance for the active realm, falling back to the root server's server-wide default knex instance.
 
 #### `server.models([all])`
-Returns an object containing models keyed by `name`.  When `all` is `true`, models across the entire server are returned.  Otherwise, only models declared within the current plugin (or, active realm) are returned.
+Returns an object containing models keyed by `name`.  When `all` is `true`, models across the entire server are returned.  Otherwise, only models declared within a.) the current plugin (or, server's active realm) and b.) any plugins for which the current plugin is an ancestor (e.g. if the current plugin has registered a plugin that registers models) are returned.
+
+
+The services that are available on this object are only those registered by server or .
```

Comment:
Dangles?

Reply:
aegrfkajwefgkewugu3qi extra dangles. sorry, slopslopslop, I'll clear that up